diff -Naur lightdm-1.10.3.orig/data/lightdm.conf lightdm-1.10.3/data/lightdm.conf
--- lightdm-1.10.3.orig/data/lightdm.conf	2014-09-11 08:07:11.000000000 +0700
+++ lightdm-1.10.3/data/lightdm.conf	2014-10-12 20:36:22.367184957 +0700
@@ -18,7 +18,7 @@
 # greeters-directory = Directory to find greeters
 #
 [LightDM]
-#start-default-seat=true
+start-default-seat=true
 #greeter-user=lightdm
 #minimum-display-number=0
 #minimum-vt=7
@@ -96,12 +96,12 @@
 #xdmcp-key=
 #unity-compositor-command=unity-system-compositor
 #unity-compositor-timeout=60
-#greeter-session=example-gtk-gnome
+greeter-session=lightdm-webkit-greeter
 #greeter-hide-users=false
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
 #greeter-show-remote-login=true
-#user-session=default
+user-session=blankon
 #allow-user-switching=true
 #allow-guest=true
 #guest-session=
